ViewModels/MainPageViewModel.cs

```
@@ -300,7 +300,10 @@ public string ArchipelagoVersion
         }
         public MainPageViewModel(string archipelagoVersion = "0.5.1")
         {
-            ConnectClickedCommand = new Command(() => { ConnectClicked?.Invoke(this, new ConnectClickedEventArgs { Host = Host, Slot = Slot, Password = Password }); });
+            ConnectClickedCommand = new Command(() => { ConnectClicked?.Invoke(this, new ConnectClickedEventArgs { 
+                Host = string.IsNullOrWhiteSpace(Host) ? "127.0.0.1:38281" : Host, 
```

would you rather have this as default text or as silent default?

---

we can make it a silent default and add a placeholderText value to the textbox

---

there already is place holder text, unless you mean something else?

https://github.com/ArsonAssassin/Archipelago.Core.MauiGUI/blob/ac698dffd2bcd3a0893d8907f91e3bc5f02b5bfe/Views/MainPage.xaml#L140

---

I mean making the placeholder text match what we're defaulting to, so "127.0.0.1:38281" instead of "Enter Host e.g. Archipelago.gg:38281"

---

gotcha fixed